src/controller/java/src/chip/devicecontroller/model/ClusterState.java

```diff
@@ -90,8 +89,7 @@ public String toString() {
                 builder.append("Event ");
                 builder.append(eventId);
                 builder.append(": ");
-                builder.append(
-                    eventState.getValue() == null ? "null" : eventState.getValue().toString());
+                builder.append(eventState.getJson().toString());
```

Comment:
Could the json value(`getJson()`) be null? It is assumed to be null when a JsonException is thrown.